packages/core/admin/admin/src/content-manager/components/DynamicTable/BulkActionsBar/index.js

```diff
@@ -43,15 +52,15 @@ const BulkActionsBar = ({
 
   return (
     <>
-      {showPublish && (
-        <>
-          <Button variant="tertiary" onClick={() => handleBulkPublish(selectedEntries)}>
-            {formatMessage({ id: 'app.utils.publish', defaultMessage: 'Publish' })}
-          </Button>
-          <Button variant="tertiary">
-            {formatMessage({ id: 'app.utils.unpublish', defaultMessage: 'Unpublish' })}
-          </Button>
-        </>
+      {showPublishButton && (
+        <Button variant="tertiary" onClick={() => handleBulkPublish(selectedEntries)}>
```

Comment:
fyi you're going to have conflicts here if my PR gets merged first, since handleBulkPublish won't exist anymore